src/main/resources/basic-file-retrievers.xml

```diff
@@ -294,5 +294,23 @@
 				<constructor-arg name="retrieverName" value="retrievers/EnsemblBioMart"/>
 			</bean>
 		</entry>
+
+		<!-- NOTE: The files retrieved from PharmacoDB and IUPHAR are *both* required to create references for PharmacoDB -->
+		<entry key="PharmacoDB">
+			<bean class="org.reactome.addlinks.dataretrieval.FileRetriever" id="PharmacoDB">
+				<property name="dataURL" value="https://raw.githubusercontent.com/BHKLAB-Pachyderm/Annotations/master/drugs_with_ids.csv" />
```

Comment:
Does this need to be changed now that they're not in beta anymore?

Reply:
That _is_ the URL to their file. Previously, it was emailed to me and I placed it in the correct directory for the file processor to pick up.

Reply:
Okay, I just wanted to double-check because a URL in raw.githubusercontent looked "beta-ish"

Reply:
I know. But... that's the URL Gangesh gave me... :roll_eyes: